site/app/views/submission/HomeworkView.php

```diff
@@ -1113,7 +1113,8 @@ private function renderVersionBox(GradedGradeable $graded_gradeable, $version_in
             'has_manual_grading' => $gradeable->isTaGrading(),
             'incomplete_autograding' => $version_instance !== null ? !$version_instance->isAutogradingComplete() : false,
             // TODO: change this to submitter ID when the MiscController uses new model
-            'user_id' => $this->core->getUser()->getId(),
+            //'user_id' => $this->core->getUser()->getId(),
+            'submitter_id' => $graded_gradeable->getSubmitter()->getId(),
```

Comment:
Should the commented-out code be removed?

Reply:
Yes, you're right. Just took care of it